pype/hosts/resolve/plugin.py

```diff
@@ -1,6 +1,92 @@
+import sys
 from avalon import api
-# from pype.hosts.resolve import lib as drlib
+from pype.hosts import resolve
 from avalon.vendor import qargparse
+from pype.api import config
+
+from Qt import QtWidgets, QtCore
+
+
+class Universal_widget(QtWidgets.QDialog):
```

Comment:
Please get rid of this. First it's based on presets -> I am now working on removing all nonchangeable presets from pype-config and you're adding another. Second it's not universal at all, plus adds more complexity without any advantage. For this cases just use hardcoded inputs. It take less code, is easier to understand and find out what is the widget really doing...